doc-Infrastructure_Migration_Solution_Guide/cfme/IMS_1.1/modules/ref_Software_compatibility_matrix.adoc

```diff
@@ -12,7 +12,7 @@
 ifdef::rhv[]
 |Red Hat Virtualization |4.2.8
 endif::rhv[]
-|CloudForms .<a|4.7.3, with CFME 5.10.3
+|CloudForms .<a|4.7.0, with the CFME 5.10.3 virtual appliance
```

Comment:
Same as above

Reply:
Answer is same as above. Definitely necessary for RHV, but not sure it's required for OSP.